target/linux/octeon/base-files/lib/preinit/03_rename_interfaces

```diff
@@ -0,0 +1,15 @@
+rename_interfaces() {
+	. /lib/functions/system.sh
+	. /lib/functions.sh
```

Comment:
From my memory it's more common to have the includes outside of the function. I'd prefer that.

Reply:
sure, moved include(s) outside of the function.